modules/programs/tealdeer.nix

```diff
@@ -0,0 +1,52 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.programs.tealdeer;
+
+  tomlFormat = pkgs.formats.toml { };
+
+  configDir = if pkgs.stdenv.isDarwin then
+    "Library/Application Support"
+  else
+    config.xdg.configHome;
+
+in {
+  meta.maintainers = [ maintainers.marsam ];
+
+  options.programs.tealdeer = {
+    enable = mkEnableOption "Tealdeer";
+
+    settings = mkOption {
+      type = tomlFormat.type;
+      default = { };
```

Comment:
```suggestion
      default = { };
      defaultText = literalExpression "{ }";
```

Reply:
Is there any particular need for the `defaultText` entry? I think the manual output should be decent even with just `default = {}`, although I haven't checked recently 🙂